### PR TITLE
ddl: fix cancel drop index error (#8504)

### PR DESCRIPTION
**Pull request by crazycs520**
### What problem does this PR solve? 
cherry-pick #8504

### Check List 

Tests

### CODE REVIEW

**Comment by crazycs520** [comment]
/run-all-tests tidb-test=release-2.0 tidb-private-test=release-2.0 tikv=release-2.0

**Comment by crazycs520** [comment]
@winkyao @zimulala PTAL

**Comment by crazycs520** [comment]
/run-all-tests tidb-test=release-2.0 tidb-private-test=release-2.0 tikv=release-2.0

File: ddl/ddl_worker.go [hunk 1]
```diff
@@ -14,6 +14,7 @@
 package ddl
 
 import (
+	"github.com/pingcap/tidb/util/schemautil"
```
Comment (zimulala) [inline ddl/ddl_worker.go]: Please put it in the right place

**Comment by crazycs520** [comment]
/run-all-tests tidb-test=release-2.0 tidb-private-test=release-2.0 tikv=release-2.0

**Comment by crazycs520** [comment]
/run-integration-compatibility-test 
tidb-test=release-2.0 tidb-private-test=release-2.0 tikv=release-2.0

**Comment by crazycs520** [comment]
/run-integration-compatibility-test
tidb-test=release-2.0 tidb-private-test=release-2.0 tikv=release-2.0

**Comment by crazycs520** [comment]
/run-all-tests tidb-test=release-2.0 tidb-private-test=release-2.0 tikv=release-2.0

**Pull request merged**